### PR TITLE
Port iotconnect azure rtos sample application to CK-RX65N board (blue PCB) with ethernet support

### PR DESCRIPTION
**Pull request by nbeck-witekio**

### CODE REVIEW

File: samples/ck-rx65n/.gitignore [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/netxduo/common/
+/netxduo/Debug/
+/netxduo/ports/
```
Comment (nik-markovic) [inline samples/ck-rx65n/.gitignore]: Rather than gitiginoring individiual directories inside a directory, it is more effective to follow [this pattern](https://github.com/avnet-iotconnect/iotc-azurertos-sdk/tree/main/samples/stm32u5) that I used for Projects directory changes nesting. 

Alternatively, you can use [this older and simpler pattern](https://github.com/avnet-iotconnect/iotc-azurertos-sdk/blob/main/samples/mimxrt1060/.gitignore), though I am not sure how well it handles adding directories. Did that one not work for you?

Having each ignored directory specifically stated makes it potentially harder to upgrade to a new baseline.

File: samples/ck-rx65n/basic-sample/.gitignore [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/trash
+/HardwareDebug/
+/output
+
+/src/smc_gen/
+
+*.o
```
Comment (nik-markovic) [inline samples/ck-rx65n/basic-sample/.gitignore:7]: Can this also be imporved by adding "anything except" (exclamation) clauses?

File: samples/ck-rx65n/basic-sample/basic-sample.elf.launch [hunk 3]
```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (nik-markovic) [inline samples/ck-rx65n/basic-sample/basic-sample.elf.launch]: In other projects, we gitignore launch configurations and let the user create their own from project templates/defaults, like in case of STM32 Debug As CortexM Appication, which generatest the launch config. Similar option exist for MCU Expresso. If there are any odd lauch config options/suggestions, I add them into the readme. Is there value in this file being checked in?

File: samples/ck-rx65n/basic-sample/include/app_config.h [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+#define IOTCONNECT_CPID ""
+#define IOTCONNECT_ENV  ""
+
+#define IOTCONNECT_DUID "" // you can supply a custom device UID, or...
+#define DUID_PREFIX "" // mac address will be appended in format 012345abcdef
+
+// if it is not defined, device certs will be used in iotconnect_app.c
+#define IOTCONNECT_SYMETRIC_KEY ""
+
+#define SAMPLE_SNTP_SERVER_NAME ""    /* SNTP Server.  */
```
Comment (nik-markovic) [inline samples/ck-rx65n/basic-sample/include/app_config.h]: Why do wehave handling of SAMPLE_SNTP_SERVER_NAME different from  other projects? Default SNTP pool from pool.ntp.org cannot be used by Avnet. We are not allowed touse it buecause we do not contribute with SNTP time servers, unless you have other information. That's why I default to time.google.com (even though it is less secure).

Default pool was also less reliable than time.google.com.

File: samples/ck-rx65n/basic-sample/include/app_config.h [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+#define WIFI_SSID ""
```
Comment (nik-markovic) [inline samples/ck-rx65n/basic-sample/include/app_config.h]: These two should be moved to the top, like in stm32* projects. The priority of WiFi config should be higher than SNTP server, which can be leaft default. If WiFi config is not there, there's no connectivity.

Do we even have WiFi on this board?

File: samples/ck-rx65n/basic-sample/src/compat.c [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+int32_t az_platform_clock_msec(int64_t* out_clock_msec)
+{
+    return 0u;
+}
+
+
+int32_t az_platform_sleep_msec(int32_t milliseconds)
+{
+    return 0u;
```
Comment (nik-markovic) [inline samples/ck-rx65n/basic-sample/src/compat.c:27]: I presume this was brought in from Azure baseline? Does this mean that we cannot sleep in the project?

File: samples/ck-rx65n/basic-sample/src/demo_printf.c [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+void demo_printf_transmit_end(void)
+{
+    tx_semaphore_put(&demo_printf_semaphore);
+}
+
+void my_sw_charput_function(char c)
+{
+    tx_mutex_get(&demo_printf_mutex, TX_WAIT_FOREVER);
+
+    R_Config_SCI5_Serial_Send(&c, 1u);
```
Comment (nik-markovic) [inline samples/ck-rx65n/basic-sample/src/demo_printf.c:29]: So just for my own clarity - this project does not redirect stdout to the console, like all other ones?

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+            free((void*) command);
+        }
+    }
+    const char *ack = iotcl_create_ack_string_and_destroy_event(data, success, message);
+    if (NULL != ack) {
+        printf("Sent OTA ack: %s\r\n", ack);
+        iotconnect_sdk_send_packet(ack);
+        free((void*) ack);
+    }
+    if (needs_ota_commit) {
```
Comment (nik-markovic) [inline samples/ck-rx65n/basic-sample/src/iotconnect_app.c]: This is leftover from copying OTA capable code. Please also fix in the cloudkit sample.

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c [hunk 9]
```diff
@@ -0,0 +1,267 @@
...
+}
+
+static void on_command(IotclEventData data) {
+	char* context = NULL;
+    char *command = iotcl_clone_command(data);
+    if (NULL != command) {
+    	char* token = strtok_r(command, " ", &context);
+    	if(!strncmp(token, "led", 3)) {
+    		token = strtok_r(NULL, " ", &context);
+    		RX65N_LED_STATE state = (token[0] == '1') ||
```
Comment (nik-markovic) [inline samples/ck-rx65n/basic-sample/src/iotconnect_app.c:87]: A bit rusty on strok, but what if there is no argument? Missing null check?

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c [hunk 10]
```diff
@@ -0,0 +1,267 @@
...
+    								(strncmp(token, "true", 4) == 0 ) ?
+    								ON : OFF;
+    		set_led(LED2, state);
+    		set_led(LED1, state);
+    		command_status(data, true, command, "LED set");
+    	}
+    	else {
```
Comment (nik-markovic) [inline samples/ck-rx65n/basic-sample/src/iotconnect_app.c]: Else needs to be in the same line as the "}"

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c [hunk 11]
```diff
@@ -0,0 +1,267 @@
...
+
+    // Optional. The first time you create a data point, the current timestamp will be automatically added
+    // TelemetryAddWith* calls are only required if sending multiple data points in one packet.
+    iotcl_telemetry_add_with_iso_time(msg, iotcl_iso_timestamp_now());
+    iotcl_telemetry_set_string(msg, "version", APP_VERSION);
+//   iotcl_telemetry_set_number(msg, "cpu", 3.123); // test floating point numbers
+    // random number 0-100, cast to int so that it removes decimals in json
+    iotcl_telemetry_set_number(msg, "random", (int)((double)rand() / (double)RAND_MAX * 100.0));
+
+    iotcl_telemetry_set_bool(msg, "button", read_user_switch());
```
Comment (nik-markovic) [inline samples/ck-rx65n/basic-sample/src/iotconnect_app.c]: I am not actually sure if Itoconnnect supports booleans for 1.0 protocol. Have we tested this? Does the data seem to make sense on the web page?

**Review by nik-markovic — CHANGES_REQUESTED** [review]

File: scripts/setup-project.sh [hunk 12]
```diff
@@ -5,7 +5,7 @@ set -e
 show_help() {
   echo "Usage: $0 <project_name>"
   echo "Available projects: stm32l4, mimxrt1060, same54xpro, " \
-      "same54xprov2, maaxboardrt, rx65ncloudkit, wfi32iot"
+      "same54xprov2, maaxboardrt, ck-rx65n, rx65ncloudkit, wfi32iot"
```
Comment (rbateman-witekio) [inline scripts/setup-project.sh]: Maybe we can add a bit here to disambiguate between the ck-rx65n and rx65ncloudkit. I don't think it would be immediately obvious which one belongs to which target. Something like:

rx65ncloudkit - Green PCB, WiFi module supported, no TSIP support
ck-rx65n - Blue PCB, Ethernet supported, TSIP support

File: samples/ck-rx65n/basic-sample/include/app_config.h (hunk 4 above)
Comment (nmatthews-witekio) [reply samples/ck-rx65n/basic-sample/include/app_config.h]: Just flagging up that different companies may have different policies. But I guess it's "just sample code".

File: samples/ck-rx65n/basic-sample/src/main.c [hunk 13]
```diff
@@ -0,0 +1,424 @@
...
+#ifndef SAMPLE_DNS_SERVER_ADDRESS
+/*#define SAMPLE_DNS_SERVER_ADDRESS     IP_ADDRESS(192, 168, 100, 1)*/
+#error "SYMBOL SAMPLE_DNS_SERVER_ADDRESS must be defined. This symbol specifies the dns server address for routing. "
+#endif /* SAMPLE_DNS_SERVER_ADDRESS */
+#else
+#define SAMPLE_IPV4_ADDRESS             IP_ADDRESS(0, 0, 0, 0)
+#define SAMPLE_IPV4_MASK                IP_ADDRESS(0, 0, 0, 0)
+
+//#define SAMPLE_DNS_SERVER_ADDRESS       IP_ADDRESS(8, 8, 8, 8)
+#define SAMPLE_DNS_SERVER_ADDRESS       IP_ADDRESS(192, 168, 1, 1)
```
Comment (nmatthews-witekio) [inline samples/ck-rx65n/basic-sample/src/main.c]: Looks odd? Does everyone have a DNS setup at 192.168.1.1

File: samples/ck-rx65n/basic-sample/src/hardware_setup.c [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+    int val;
+
+    /* Setup SCI5 for printf output. */
+    R_Config_SCI5_Start();
+
+    /* Setup user switch and LEDs*/
+    R_Config_PORT_Create();
+
+    /* Create periodic timer for the system tick. */
+    R_CMT_CreatePeriodic(TX_TIMER_TICKS_PER_SECOND, timer_callback, &chan);
```
Comment (nmatthews-witekio) [inline samples/ck-rx65n/basic-sample/src/hardware_setup.c:42]: Should rx65ncloudkit hardware_setup.c also use TX_TIMER_TICKS_PER_SECOND ??

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c [hunk 15]
```diff
@@ -0,0 +1,267 @@
...
+
+//    sensors_add_telemetry(msg);
+    str = iotcl_create_serialized_string(msg, false);
+    iotcl_telemetry_destroy(msg);
+    printf("Sending: %s\r\n", str);
+    if (NULL != str) {
```
Comment (nmatthews-witekio) [inline samples/ck-rx65n/basic-sample/src/iotconnect_app.c:138]: Should probably back port to rx65ncloudkit -- so that publish_telemetry() is the same for both?

File: samples/ck-rx65n/basic-sample/src/main.c [hunk 16]
```diff
@@ -0,0 +1,424 @@
...
+
+/* Define sample helper thread entry.  */
+void sample_helper_thread_entry(ULONG parameter)
+{
+UINT    status;
+ULONG   ip_address = 0;
+ULONG   network_mask = 0;
+ULONG   gateway_address = 0;
+ULONG   dns_server_address[3];
+int val;
```
Comment (nmatthews-witekio) [inline samples/ck-rx65n/basic-sample/src/main.c]: "val" looks to be unused?

File: scripts/setup-project.sh [hunk 17]
```diff
@@ -126,6 +126,13 @@ create_threadx_projects() {
...
+    ck-rx65n)
+      echo Downloading Azure_RTOS_6...
+      wget -q -O azrtos.zip https://saleshosted.z13.web.core.windows.net/sdk/AzureRTOS/Azure_RTOS_6.1_RX65N_Cloud_Kit_E2Studio_GNURX_Samples_2022_05_25.zip
+      project_platform_dir=''
+      project_ide_dir='e2studio_gnurx/'
+      libs="filex netxduo_addons "
+      ;;
     rx65ncloudkit)
       echo Downloading Azure_RTOS_6...
       wget -q -O azrtos.zip https://saleshosted.z13.web.core.windows.net/sdk/AzureRTOS/Azure_RTOS_6.1_RX65N_Cloud_Kit_E2Studio_GNURX_Samples_2022_05_25.zip
```
Comment (nmatthews-witekio) [inline scripts/setup-project.sh]: Crossing patches -- need to use ${FETCH} instead of "wget -q -O" when rebase / merge back to main.
See recent changes to setup-project.sh

File: scripts/setup-project.sh [hunk 18]
```diff
@@ -171,7 +178,7 @@ create_threadx_projects() {
       sed -i 's#nxd#netxduo#g' ./netxduo/.cproject
       sed -i 's#tx#threadx#g' ./threadx/.cproject
       ;;
-    mimxrt1060 | rx65ncloudkit)
+    mimxrt1060 | ck-rx65n | rx65ncloudkit)
```
Comment (nmatthews-witekio) [inline scripts/setup-project.sh:214]: Probably missing something - does ck-rx65n need an "ethernet library" -- not sure I can find r_ether_rx_if.h
It's probably me -- but just in case.

File: samples/ck-rx65n/basic-sample/src/main.c [hunk 19]
```diff
@@ -0,0 +1,424 @@
...
+        printf("Demo helper thread creation fail: %u\r\n", status);
+        return;
+    }
+}
+
+/* Define sample helper thread entry.  */
+void sample_helper_thread_entry(ULONG parameter)
+{
+UINT    status;
+ULONG   ip_address = 0;
```
Comment (nmatthews-witekio) [inline samples/ck-rx65n/basic-sample/src/main.c:271]: shadows a global

File: samples/ck-rx65n/basic-sample/src/main.c [hunk 20]
```diff
@@ -0,0 +1,424 @@
...
+ULONG   network_mask = 0;
```
Comment (nmatthews-witekio) [inline samples/ck-rx65n/basic-sample/src/main.c:272]: shadows a global

File: samples/ck-rx65n/basic-sample/src/main.c [hunk 21]
```diff
@@ -0,0 +1,424 @@
...
+ULONG   gateway_address = 0;
+ULONG   dns_server_address[3];
```
Comment (nmatthews-witekio) [inline samples/ck-rx65n/basic-sample/src/main.c]: unused?

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c [hunk 22]
```diff
@@ -0,0 +1,267 @@
...
+        break;
+    }
+    if (NULL != auth_driver_context) {
+    	release_sw_der_auth_driver(auth_driver_context);
+    }
+}
+
+static void publish_telemetry() {
+    const char *str;
+    IotclMessageHandle msg = iotcl_telemetry_create(iotconnect_sdk_get_lib_config());
```
Comment (nmatthews-witekio) [inline samples/ck-rx65n/basic-sample/src/iotconnect_app.c]: iotcl_telemetry_create() doesn't take arguments

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c [hunk 23]
```diff
@@ -0,0 +1,267 @@
...
+//   iotcl_telemetry_set_number(msg, "cpu", 3.123); // test floating point numbers
+    // random number 0-100, cast to int so that it removes decimals in json
+    iotcl_telemetry_set_number(msg, "random", (int)((double)rand() / (double)RAND_MAX * 100.0));
+
+    iotcl_telemetry_set_bool(msg, "button", read_user_switch());
+
+//    sensors_add_telemetry(msg);
+    str = iotcl_create_serialized_string(msg, false);
+    iotcl_telemetry_destroy(msg);
+    printf("Sending: %s\r\n", str);
```
Comment (nmatthews-witekio) [inline samples/ck-rx65n/basic-sample/src/iotconnect_app.c:137]: Move printf inside the `if (NULL != str) {` block - to avoid NULL dereference.

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c [hunk 24]
```diff
@@ -0,0 +1,267 @@
...
+bool app_startup(NX_IP *ip_ptr, NX_PACKET_POOL *pool_ptr, NX_DNS *dns_ptr) {
+    printf("Starting App Version %s\r\n", APP_VERSION);
+    IotConnectClientConfig *config = iotconnect_sdk_init_and_get_config();
+    azrtos_config.ip_ptr = ip_ptr;
+	azrtos_config.pool_ptr = pool_ptr;
+	azrtos_config.dns_ptr = dns_ptr;
+
+    config->cpid = IOTCONNECT_CPID;
+    config->env = IOTCONNECT_ENV;
+#ifdef IOTCONNECT_DUID
```
Comment (nmatthews-witekio) [inline samples/ck-rx65n/basic-sample/src/iotconnect_app.c:154]: Unfair to raise it on this specific branch, but...

If 
`#define IOTCONNECT_DUID "" `
don't think this `#ifdef` test works?

Probably need a run-time test (C preprocessor not easy to find empty strings?).

File: samples/ck-rx65n/basic-sample/src/main.c (hunk 13 above)
Comment (nik-markovic) [reply samples/ck-rx65n/basic-sample/src/main.c]: @nmatthews-witekio  I think that in most other projects even though this is defined, it's not used because DNS server is obtained from DHCP negotiation. Those are maybe some sort of defaults to fallback id DHCP fails? You can maybe verify that in the log output. It should print the obtained DNS address from DHCP.

You will seee #endif /* SAMPLE_DHCP_DISABLE */ at the end of that block. We do not do SAMPLE_DHCP_DISABLE in any project, so thase end up being not defined - as intended.

The point of not elliminating those defines alltogher is to try to have as little as possible diff agains the main.c provided by Microsoft.

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c (hunk 24 above)
Comment (nik-markovic) [reply samples/ck-rx65n/basic-sample/src/iotconnect_app.c:154]: @nmatthews-witekio I think that that code means to say if you did not define DUID, you want to use dynamically generated DUID (not all projects actually do this). If you define a blank one, it will abviously not work. We don't need to check for blank.

FYI, this works with L4 - obtained from WiFi MAC, I think and maybe imx 1060, I doubt that this dynamic generation in compose_device_id() works on any other project. I think it's a dummy DUID_PREFIX000000000 for all others, so the customer can plug in that missing part. In this project, you will see that it declares an empty mac_addr array and does not poppulate it before it assigns a dynamic name.

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c (hunk 23 above)
Comment (nik-markovic) [reply samples/ck-rx65n/basic-sample/src/iotconnect_app.c:137]: @nmatthews-witekio  Not saying that this is correct, but what do we do if it indeed is null? This is not some transient thing that we can work around with - sending telemtry is the purpose of the demo. Null is not returned ever by that function unless user messed up something or we ran out of RAM. Probably better to crash than silently ignore it. I've never seen us run out of ram after sending the first batch of telemetry. I also tested for memory leaks and fragmentation in the sample (up to a degree) and did not find issues.

Issue is inherited from all demos. Maybe let's not burden Nick with fixing this. We can do a ticket and I can go out and fix it in all projects.

File: samples/ck-rx65n/basic-sample/src/main.c (hunk 20 above)
Comment (nik-markovic) [reply samples/ck-rx65n/basic-sample/src/main.c:272]: this code:
```C
#ifndef SAMPLE_DHCP_DISABLE
static NX_DHCP          dhcp_0;
NX_DHCP           dhcp_client;
UCHAR             ip_address[4];
UCHAR             network_mask[4];
#endif /* SAMPLE_DHCP_DISABLE  */
```
FYI, I'm curious where it came from. I searched for this UCHAR declaration and I cannot find it in any of the mains from Microsoft's Azure_RTOS_6.1_RX65N_Cloud_Kit_E2Studio_GNURX_Samples_2022_05_25.zip.

SAMPLE_DHCP_DISABLE should not be defined, so these declarations are indeed being shadowed.

File: samples/ck-rx65n/.gitignore (hunk 1 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/.gitignore]: Updated

File: samples/ck-rx65n/basic-sample/basic-sample.elf.launch (hunk 3 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/basic-sample.elf.launch]: Removed and instructions added to README.md for launcher creation

File: samples/ck-rx65n/basic-sample/include/app_config.h (hunk 4 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/include/app_config.h]: Resolved

File: samples/ck-rx65n/basic-sample/include/app_config.h (hunk 5 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/include/app_config.h]: Removed from header file as not applicable

File: samples/ck-rx65n/basic-sample/src/demo_printf.c (hunk 7 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/src/demo_printf.c:29]: printf() stdout is routed to the console, this is how it is implemented in the code from Renesas/Microsoft

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c (hunk 8 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/src/iotconnect_app.c]: Removed OTA related bits and return a not implemented message

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c (hunk 9 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/src/iotconnect_app.c:87]: If there is no argument it will turn the LED off

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c (hunk 10 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/src/iotconnect_app.c]: Done

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c (hunk 11 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/src/iotconnect_app.c]: It does seem to work correctly as a boolean for live data tabular and  telemetry pages, but the graph doesn't render the bool and the SDK documentation and templeate boolean are not mentioned for v1.0 so have changed it to be a number (as this is what the template says too technically)

File: scripts/setup-project.sh (hunk 12 above)
Comment (nbeck-witekio) [reply scripts/setup-project.sh]: Done

File: samples/ck-rx65n/basic-sample/src/main.c (hunk 16 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/src/main.c]: Removed unused variables

File: scripts/setup-project.sh (hunk 17 above)
Comment (nbeck-witekio) [reply scripts/setup-project.sh]: Done - Rebased on main

File: scripts/setup-project.sh (hunk 18 above)
Comment (nbeck-witekio) [reply scripts/setup-project.sh:214]: Should be automatically generated code when built

File: samples/ck-rx65n/basic-sample/src/main.c (hunk 19 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/src/main.c:271]: Removed shadowed vars

File: samples/ck-rx65n/basic-sample/src/main.c (hunk 21 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/src/main.c]: Done

File: samples/ck-rx65n/basic-sample/src/iotconnect_app.c (hunk 22 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/src/iotconnect_app.c]: Done

File: samples/ck-rx65n/basic-sample/src/compat.c (hunk 6 above)
Comment (nbeck-witekio) [reply samples/ck-rx65n/basic-sample/src/compat.c:27]: This was from the Azure baseline

File: scripts/setup-project.sh [hunk 25]
```diff
@@ -159,6 +159,13 @@ create_threadx_projects() {
       project_ide_dir='mplab/'
       libs="same54_lib filex common_hardware_code "
       ;;
+    ck-rx65n)
+      echo Downloading Azure_RTOS_6...
+      wget -q -O azrtos.zip https://saleshosted.z13.web.core.windows.net/sdk/AzureRTOS/Azure_RTOS_6.1_RX65N_Cloud_Kit_E2Studio_GNURX_Samples_2022_05_25.zip
```
Comment (nmatthews-witekio) [inline scripts/setup-project.sh:164]: Need to use ${FETCH} rather than wget here.

Comment (nmatthews-witekio) [reply scripts/setup-project.sh:164]: See https://github.com/avnet-iotconnect/iotc-azurertos-sdk/pull/48

Comment (nbeck-witekio) [reply scripts/setup-project.sh:164]: I'd made that change in https://github.com/avnet-iotconnect/iotc-azurertos-sdk/pull/47/files#diff-42bd5a8d71dda5434de8eb708560bad0c8f448068c38766613305898b27f35f8 which is branched from this branch, but not in here. Could merge PR #48 and rebase PR#47 on top of that if that is desired?

File: samples/ck-rx65n/basic-sample/.gitignore (hunk 2 above)
Comment (nik-markovic) [reply samples/ck-rx65n/basic-sample/.gitignore:7]: I will resolve this so we can have a baseline on main. If there is any other work, we can do that later.

**Review by nik-markovic — APPROVED** [review]
